src/pages/Login/index.jsx

```diff
@@ -1,3 +1,183 @@
+"use client";
+import React from "react";
+import { useNavigate } from "react-router-dom";
+
+//for field validation
+import { object, string } from "yup";
+
+//nextUI components
+import { Image, Input, Button } from "@nextui-org/react";
+import { EyeFilledIcon } from "./EyeFilledIcon";
+import { EyeSlashFilledIcon } from "./EyeSlashFilledIcon";
+
+import authBackground from "../../assets/authBackground.png";
+
+let userSchema = object({
+  email: string()
+    .required("Username is required")
+    .matches(
+      /^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,6}$/,
+      "Invalid Email Format"
+    )
+    .label("email"),
+  password: string().required("Password Is Required").label("password"),
+});
+
 export const LoginPage = () => {
-  return <p>Job Quest Login</p>;
+  const [userInfo, setUserInfo] = React.useState({
+    email: "",
+    password: "",
+  });
+
+  //used to shwo or not show password.
+  const [isVisible, setIsVisible] = React.useState(false);
+
+  //form loading
+  const [isLoading, setIsLoading] = React.useState(false);
+  const [errors, setErrors] = React.useState({});
+
+  //Router navigation
+  const navigateTo = useNavigate();
+
+  React.useEffect(() => {
+    if (userInfo.email === "" || userInfo.password === "") {
+      setErrors((prevErrors) => ({
+        ...prevErrors,
+        email: "",
+        password: "",
+      }));
+    }
+  }, [userInfo]);
+
+  async function handleSubmit() {
+    try {
+      setIsLoading(true);
+      let result = await userSchema.validate(userInfo, { abortEarly: false });
+
+      const email = localStorage.getItem("jobQuestEmail");
+      const password = localStorage.getItem("jobQuestPassword");
+
+      //if the result email and password field are equal to values saved in localStorage, then the user is authenticated.
+      if (email === result.email && password === result.password) {
+        navigateTo("../Roadmap");
```

Comment:
this should be changed to `navigateTo("/roadmap")` - as defined by Poorva in her own changes.